code/AssetLib/glTF2/glTF2Asset.inl

```diff
@@ -1628,6 +1628,22 @@ inline void Node::Read(Value &obj, Asset &r) {
                 }
             }
         }
+
+        // Read the the nodes specified under MSFT_lod extension ids and save as children node
+        if (Value *msftLodExt = FindObject(*curExtensions, "MSFT_lod")) {
```

Comment:
Do we need a line like https://github.com/Esri/assimp/blob/c7c71a0b5a4cf8b383b447fb95f959887644f2ff/code/AssetLib/glTF2/glTF2Asset.inl#L1620? Maybe that ties into what @qjk-redlands is asking about as well?

Reply:
There is no way to find this info. Although glTF json has `"extensionsUsed": ["MSFT_lod"]` this property but this is only used to check the core extensions defined in the list